Groups cannot have binding configurations for 1.x bindings

In openHAB 1.x also groups could have bindings which seem not to be working any more in openHAB 2.0.

The follwing item definition worked fine in openHAB 1.x
`Group:Switch:OR(ON,OFF) gPresence	"Presence [MAP(presence_display.map):%s]"		{mqtt=">[broker:presence/status:command:*:MAP(presence_out.map)]"}`

In openHAB 2.0 the binding is not accepted and the logs give
```
2017-01-25 13:44:03.630 [ERROR] [el.item.internal.GenericItemProvider] - Binding configuration of type 'mqtt' of item 'gPresence' could not be parsed correctly.
org.eclipse.smarthome.model.item.BindingConfigParseException: cannot process unknown item type Group
	at org.openhab.core.binding.internal.BindingConfigReaderDelegate.getOpenHABItem(BindingConfigReaderDelegate.java:84)[176:org.openhab.core.compat1x:2.0.0]
	at org.openhab.core.binding.internal.BindingConfigReaderDelegate.getOpenHABItem(BindingConfigReaderDelegate.java:56)[176:org.openhab.core.compat1x:2.0.0]
	at org.openhab.core.binding.internal.BindingConfigReaderDelegate.validateItemType(BindingConfigReaderDelegate.java:37)[176:org.openhab.core.compat1x:2.0.0]
	at org.eclipse.smarthome.model.item.internal.GenericItemProvider.internalDispatchBindings(GenericItemProvider.java:311)[123:org.eclipse.smarthome.model.item:0.9.0.b4]
	at org.eclipse.smarthome.model.item.internal.GenericItemProvider.internalDispatchBindings(GenericItemProvider.java:284)[123:org.eclipse.smarthome.model.item:0.9.0.b4]
	at org.eclipse.smarthome.model.item.internal.GenericItemProvider.processBindingConfigsFromModel(GenericItemProvider.java:167)[123:org.eclipse.smarthome.model.item:0.9.0.b4]
	at org.eclipse.smarthome.model.item.internal.GenericItemProvider.modelChanged(GenericItemProvider.java:347)[123:org.eclipse.smarthome.model.item:0.9.0.b4]
	at org.eclipse.smarthome.model.core.internal.ModelRepositoryImpl.notifyListeners(ModelRepositoryImpl.java:207)[122:org.eclipse.smarthome.model.core:0.9.0.b4]
	at org.eclipse.smarthome.model.core.internal.ModelRepositoryImpl.addOrRefreshModel(ModelRepositoryImpl.java:120)[122:org.eclipse.smarthome.model.core:0.9.0.b4]
	at org.eclipse.smarthome.model.core.internal.folder.FolderObserver.checkFile(FolderObserver.java:272)[122:org.eclipse.smarthome.model.core:0.9.0.b4]
	at org.eclipse.smarthome.model.core.internal.folder.FolderObserver.access$1(FolderObserver.java:265)[122:org.eclipse.smarthome.model.core:0.9.0.b4]
	at org.eclipse.smarthome.model.core.internal.folder.FolderObserver$WatchQueueReader.processWatchEvent(FolderObserver.java:146)[122:org.eclipse.smarthome.model.core:0.9.0.b4]
	at org.eclipse.smarthome.core.service.AbstractWatchQueueReader.run(AbstractWatchQueueReader.java:122)[98:org.eclipse.smarthome.core:0.9.0.b4]
	at java.lang.Thread.run(Thread.java:745)[:1.8.0_111]
```

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/41339931-groups-cannot-have-bindings-in-openhab-2-0?utm_campaign=plugin&utm_content=tracker%2F28054058&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F28054058&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/bundles/org.openhab.core.compat1x/src/main/java/org/openhab/core/binding/internal/BindingConfigReaderDelegate.java
+++ b/bundles/org.openhab.core.compat1x/src/main/java/org/openhab/core/binding/internal/BindingConfigReaderDelegate.java
@@ -60,6 +60,8 @@ public class BindingConfigReaderDelegate implements BindingConfigReader {
             throws BindingConfigParseException {
 
         switch (itemType) {
+            case "Group":
+                return new org.openhab.core.items.GroupItem(itemName);
             case "Switch":
                 return new org.openhab.core.library.items.SwitchItem(itemName);
             case "Dimmer":

